Detectors/FIT/FT0/reconstruction/include/FT0Reconstruction/CTFCoder.h

```diff
@@ -141,28 +147,33 @@ void CTFCoder::decompress(const CompressedDigits& cd, VDIG& digitVec, VCHAN& cha
     firstEntry = channelVec.size();
     uint8_t chID = 0;
     int amplA = 0, amplC = 0, timeA = 0, timeC = 0;
+    int mTime_trg_gate = 192; // #channels
     for (uint8_t ic = 0; ic < cd.nChan[idig]; ic++) {
       auto icc = channelVec.size();
-      const auto& chan = channelVec.emplace_back((chID += cd.idChan[icc]), cd.cfdTime[icc], cd.qtcAmpl[icc], cd.qtc[icc]);
+      const auto& chan = channelVec.emplace_back((chID += cd.idChan[icc]), cd.cfdTime[icc], cd.qtcAmpl[icc], cd.qtcChain[icc]);
       //
       // rebuild digit
       if (chan.ChId < 4 * uint8_t(Geometry::NCellsA)) { // A side
-        trig.nChanA++;
-        amplA += chan.QTCAmpl;
-        timeA += chan.CFDTime;
+        if (std::abs(chan.CFDTime < mTime_trg_gate)) {
```

Comment:
It will be faster to have A/C channel determination inside the the time comparison if:
````
if (std::abs(chan.CFDTime < mTime_trg_gate)) {
   if (chan.ChId < 4 * uint8_t(Geometry::NCellsA)) { // A side
     ...
   }
   else {
      ....
   }
}
````